aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/translator/resourcemodel/ReadResourceModelTranslator.java

```diff
@@ -59,4 +63,30 @@ public static ResourceModel translateToResourceModel(final GetPatchBaselineRespo
         return model;
     }
 
+    /**
+     * Translate Read Response to Resource Model
+     */
+    public static ResourceModel translateToResourceModel(final DescribePatchBaselinesResponse describePatchBaselinesResponse,
```

Comment:
what is tags for?

Reply:
Removed the tags parameter since we are not using Tags details in describe patch baseline